spcs/user_metrics/template-1/Dockerfile.consumer.prometheus

```diff
@@ -0,0 +1,5 @@
+# Use the official Prometheus image as the base image
```

Comment:
how would this work without the plugin?

Reply:
plugin is needed for otel
our stack is otel/prom/grafana